lib/ansible/plugins/action/reboot.py

```diff
@@ -197,8 +198,17 @@ def perform_reboot(self):
         display.debug("%s: rebooting server" % self._task.action)
 
         remote_command = self.construct_command()
-        reboot_result = self._low_level_execute_command(remote_command, sudoable=self.DEFAULT_SUDOABLE)
+
         result = {}
+        reboot_result = {}
+
+        # Catch connections that are severed too quickly and handle it gracefully
```

Comment:
I'd put the comment in the exception handler instead of at the top of the entire block.